tests/common/snappi/snappi_helpers.py

```diff
@@ -238,3 +239,73 @@ def get_dut_port_id(dut_hostname, dut_port, conn_data, fanout_data):
            return i
 
     return None
+
+
+def wait_for_arp(snappi_api, max_attempts=10, poll_interval_sec=1):
+    """
+    This function waits for arp to get resolved for ipv4/ipv6 in the snappi config
+
+    Args:
+    snappi_api: snappi api
+    max_attempts: maximum attempts for timeout
+    poll_interval_sec: interval poll second
+
+    Return:
+    returns number of attempts if arp is resolved within max attempts else fail 
+    """
+    attempts = 0
+    arp = False
+    nd = False
```

Comment:
What is nd?

Reply:
neighbor discovery for ipv6

Reply:
> neighbor discovery for ipv6

Is neighbor discovery equivalent to ARP resolved? If yes, can we use arp_ipv4 and arp_ipv6 as variable names?

Reply:
yeah, agree..since the function name has only arp, I will update the PR with variable names to arp_ipv4 & arp_ipv6